packages/altair-app/src/app/store/settings/settings.reducer.ts

```diff
@@ -9,75 +9,74 @@ export type SettingsLanguage = keyof typeof config.languages;
 export interface State {
 
   /**
-   * Specifies the theme
-   * Options: 'light', 'dark', 'dracula'
+   * Theme
    */
   theme: string;
 
   /**
-   * Specifies the language e.g. 'en-US', 'fr-FR', 'ru-RU', etc
+   * Set language
    */
   language: SettingsLanguage;
 
   /**
-   * Specifies how deep the 'Add query' functionality would go
+   * 'Add query' functionality depth
    */
   addQueryDepthLimit: number;
 
   /**
-   * Specifies the tab size in the editor
+   * Editor tab size
    */
   tabSize: number;
 
   /**
-   * Enable experimental features in Altair.
-   * Note: The features might be unstable
+   * Enable experimental features.
+   * Note: Might be unstable
    */
   enableExperimental?: boolean;
 
   /**
-   * Specifies the base font size
-   * (Default size - 24)
+   * Base Font Size
+   * (Default - 24)
    */
   'theme.fontsize'?: number;
 
   /**
-   * Specifies the font family for the editors
+   * Font Family
```

Comment:
This should indicate that it's the font family for the editor right? It doesn't change the font family for the interface